spec/models/viewed_secret_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 describe ViewedSecret do
+  it 'must have an account' do
+    viewed_secret = described_class.create(:account => nil).should be_invalid
```

Comment:
Nice test! Super minor thing, but in situations like this you can use `build` instead of `create` and the test is a little faster because it doesn't hit the DB.